docs/Manifest.toml

```diff
@@ -1,37 +1,63 @@
+# This file is machine-generated - editing it directly is not advised
+
 [[Base64]]
 uuid = "2a0f44e3-6c83-55bd-87e4-b1978d98bd5f"
 
+[[CRlibm]]
+deps = ["Libdl"]
+git-tree-sha1 = "9d1c22cff9c04207f336b8e64840d0bd40d86e0e"
+uuid = "96374032-68de-5a5b-8d9e-752f78720389"
+version = "0.8.0"
+
 [[Dates]]
 deps = ["Printf"]
 uuid = "ade2ca70-3891-5945-98fb-dc099432e06a"
 
 [[Distributed]]
-deps = ["LinearAlgebra", "Random", "Serialization", "Sockets"]
+deps = ["Random", "Serialization", "Sockets"]
 uuid = "8ba89e20-285c-5b6f-9357-94700520ee1b"
 
 [[DocStringExtensions]]
 deps = ["LibGit2", "Markdown", "Pkg", "Test"]
-git-tree-sha1 = "4d30e889c9f106a51ffa4791a88ffd4765bf20c3"
+git-tree-sha1 = "88bb0edb352b16608036faadcc071adda068582a"
 uuid = "ffbed154-4ef7-542d-bbb7-c09d3a79fcae"
-version = "0.7.0"
+version = "0.8.1"
```

Comment:
Are you sure that these are the minimum possible versions? E.g. 0.8.0 doesn't work? Probably this should just be 0.8

Reply:
I don't think the Manifest.toml file should actually be included at all?

Reply:
I'm not completely sure about this:
- I think the Manifest.toml is useful in the `docs` such that travis/GitHub Actions use the same version every time to generate the docs. Otherwise you need to provide upper bounds to be sure that it doesn't break when Documenter updates. 
Users of a package normally don't generate the docs locally. (At least I don't :smile: ) Therefore I think the versions used there are not breaking anything for the user.
The root Manifest.toml shouldn't be included I think because there it should use the highest version possible if allowed by the specified `[Compat]` section and also a strict version shouldn't be forced as long as the package works the version for subpackages is often not important.

Reply:
OK thanks for the comments. Shall I just go ahead and merge this then? Or will the docs be built during this PR?

Reply:
Unfortunately I think you need to merge this to see the actual output. 
I'm not sure why travis is not running for this package. Do you only run it on master?

Reply:
Seems like for the other PRs travis is running :confused:

Reply:
I expect that's out of date too.

Reply:
I'll just go ahead and merge.

Reply:
Maybe you get some useful information on travis.org?

Reply:
It did run:
https://travis-ci.org/github/JuliaIntervals/IntervalConstraintProgramming.jl/builds/670175731

Don't know why that wasn't picked up.